### PR TITLE
LPS-177817 Add condition for rendering to Lead List

### PR DESCRIPTION
**Pull request by henriquelimadotcom**
https://issues.liferay.com/browse/LPS-177817

The behavior was expected, however it was enough to make a change so that the "List of Leads" tab would not appear when selecting "event" in "Type of Activity"

### CODE REVIEW

**Comment by henriquelimadotcom** [comment]
ci:test:sf

**Comment by henriquelimadotcom** [comment]
ci:test:sf

**Comment by henriquelimadotcom** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestForm/steps/Activities/components/Form/components/LeadListSection/LeadListSection.tsx [hunk 1]
```diff
@@ -59,19 +59,17 @@ const LeadListSection = ({
 		}
 	};
 
-	return (
+	return selectedTypeActivity !== TypeActivityKey.EVENT ? (
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestForm/steps/Activities/components/Form/components/LeadListSection/LeadListSection.tsx:62]: Hey @henriquelimadotcom. if the selectedTypeActivity is required to show the entired component, the best is to do this check outside this component.

Also, it's better to do using && instead ternary.

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Comment by henriquelimadotcom** [comment]
ci:test:sf

**Review by jairmedeiros — APPROVED** [review]

**Comment by jairmedeiros** [comment]
ci:forward

**Comment by henriquelimadotcom** [comment]
ci:test:sf

**Comment by jairmedeiros** [comment]
ci:forward

**Pull request closed without merging**